src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -53,7 +53,11 @@ impl CompactionPicker for LevelCompactionPicker {
         }
 
         if self.config.split_by_state_table {
-            let mut member_table_ids = levels.member_table_ids.clone();
+            let mut member_table_ids = levels
+                .member_tables
+                .iter()
```

Comment:
why not clone it directly

Reply:
> why not clone it directly

We can clone it directly in the future for further optimization.